cmd/batch-handlers.go

```diff
@@ -1116,9 +1116,11 @@ func (r *BatchJobReplicateV1) Validate(ctx context.Context, job BatchJobRequest,
 	if r.Source.Bucket == "" {
 		return errInvalidArgument
 	}
+	var isRemoteToLocal bool
 	localBkt := r.Source.Bucket
 	if r.Source.Endpoint != "" {
 		localBkt = r.Target.Bucket
+		isRemoteToLocal = true
```

Comment:
could the below error  (line 1130) be updated to match source/target because the error message is not intuitive ?,  if the bucket is not present in target. 
it  would also be helpful to include the bucket name as part of the error message.

Reply:
@poornas ^^